tests/parser/types/numbers/test_uint256.py

```diff
@@ -1,17 +1,44 @@
-# from ethereum.abi import ValueOutOfBounds
+from vyper.exceptions import (
+    InvalidLiteralException
+)
 
 
-def test_convert_bytes_to_uint256(get_contract_with_gas_estimation):
-    test_contract = """
+def test_convert_bytes_to_uint256(assert_compile_failed, get_contract_with_gas_estimation):
+    # Test valid bytes input for conversion
+    test_success = """
 @public
 def foo(bar: bytes[5]) -> uint256:
     return convert(bar, uint256)
     """
 
-    c = get_contract_with_gas_estimation(test_contract)
+    c = get_contract_with_gas_estimation(test_success)
     assert c.foo(b'\x00\x00\x00\x00\x00') == 0
     assert c.foo(b'\x00\x07\x5B\xCD\x15') == 123456789
 
+    # Test overflow bytes input for conversion
+    test_fail = """
+@public
+def foo(bar: bytes[40]) -> uint256:
```

Comment:
Change this to 33. Always test the edge cases :)

Reply:
Ah yes sorry will do, also will do on #1085

Reply:
😅

Reply:
@fubuloubu Added better tests!

Reply:
Wait, is #1085 mixed commits with this one?

Reply:
Ah yes, I missed the fact that here you are testing uint256, and in the other one you are testing int128